ompi/mca/coll/han/coll_han_dynamic.c

```diff
@@ -304,6 +305,60 @@ get_module(COLLTYPE_T coll_id,
     return han_module->modules_storage.modules[mca_rule_component].module_handler;
 }
 
+/*
+ * whether OMPI_MCA_coll_han_use_xxx_algorithm was set by user
+ */
+static bool
+han_algorithm_is_user_provided(COLLTYPE_T coll_id) {
+    const int *value = NULL;
+    mca_base_var_source_t source;
+    mca_base_var_get_value(mca_coll_han_component.use_algorithm_param[coll_id],
+                           &value, &source, NULL);
+    return (MCA_BASE_VAR_SOURCE_DEFAULT != source);
+}
+
+/*
+ * Return the algorithm to use for the collective coll_id
+ * for a msg_size sized message on the comm communicator
+ * following the dynamic rules
+ */
+static int
+get_algo(COLLTYPE_T coll_id,
```

Comment:
Can we change the name of this function please (I know `algo` make sense in our language but that's not universal).

Reply:
I changed to get_algorithm
I changed in coll_han_algorithm.h/.c too